.github/workflows/preview-release-on-comment.yml

```diff
@@ -49,20 +49,21 @@ jobs:
         uses: actions/checkout@v4
         with:
           ref: ${{ steps.comment-branch.outputs.head_ref }}
-          token: ${{ steps.generate_github_token.outputs.token }}   
-
-      - name: Setup Node.js
-        uses: actions/setup-node@v3
-        with:
-          node-version: 18
```

Comment:
We don't want to hardcode it.

Installing node is already done in the `./.github/actions/ci` action